content/git/concepts/fork/fork.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Fork'
+Description: 'Making a duplicate of a repository to work on separately is known as forking.'
+Subjects: 
+  - 'Code Foundations'
+  - 'Developer Tools'
+  - 'Web Development'
+Tags:
+  - 'Git'
+  - 'GitHub'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-git'
+  - 'learn-the-command-line'
+---
+
+**Fork** is making a duplicate of an existing repository. This "fork" becomes a separate undertaking that you are in charge of. Collaboration, experimentation, and contributions to open-source code are made possible by forking. It enables developers worldwide to collaborate, make changes, and contribute to projects while retaining the integrity of the source code. It is a secure approach to modify and test code without changing the original.
+
+## Steps to Fork a Repository
+
+### Step 1
+
+First, Login to your GitHub account.
+
+### Step 2
+
+Navigate to the Github repository you want to fork.
+
+### Step 3
+
+Click the "Fork" button in the top right corner.
+
+![Repo page with Fork button](https://i.imgur.com/000HPJD.png)
```

Comment:
If you want to put an image here, follow the instructions on hosting media in the style guide [here](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md#file-hosting).